src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -98,7 +100,33 @@ export function setupTerminalMenus(): void {
 					order: 4,
 					when: TerminalContextKeys.processSupported
 				}
-			}
+			},
+			{
+				id: MenuId.ViewTitle,
+				item: {
+					command: {
+						id: TerminalCommandId.StartVoice,
+						title: localize('workbench.action.terminal.startVoice', "Start Dictation"),
+					},
+					group: TerminalMenuBarGroup.Voice,
```

Comment:
I think these living in the voice group, not `'navigation'`, means it won't support right click to show it for terminal view specifically?

Reply:
Oh, TIL!

Reply:
@meganrogge that's the tab menu, you want to right click on the view actions:

<img width="260" height="370" alt="image" src="https://github.com/user-attachments/assets/69b38aa1-cc4a-4b5c-b3b8-434408f0eb8d" />

> I also think it looks better in a separate group

It does look better, but it defeats the purpose imo if you cannot toggle it to show always as it's not very convenient in the overflow.

Reply:
ah, in that case we need Codicons for the actions too.